src/Diagnostics/FxCop/Desktop.Analyzers/CSharp/CSharpDesktopAnalyzers.csproj

```diff
@@ -0,0 +1,90 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Label="Settings">
+    <Import Project="..\..\..\..\Tools\Microsoft.CodeAnalysis.Toolset.Open\Targets\VSL.Settings.targets" />
+    <Import Project="..\..\..\..\..\build\VSL.Settings.Closed.targets" />
+  </ImportGroup>
+  <PropertyGroup>
+    <MinimumVisualStudioVersion>12.0</MinimumVisualStudioVersion>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{9A61D22F-A414-47C5-A72A-6546ACECDB23}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AnalyzerProject>true</AnalyzerProject>
+    <RootNamespace>Desktop.CSharp.Analyzers</RootNamespace>
+    <AssemblyName>Desktop.CSharp.Analyzers</AssemblyName>
```

Comment:
Sure, i'll be happy to find a better name. Analyzers in this package are meant to me related to API that do not appear in any contract assemblies but are a part of desktop .net framework and friends. For example, rule CS2153 fires when a HandleCorruptedStateExceptionAttribute is present, which cannot happen in the contract world. We had a discussion with Sri about it and decided to separate such rules in Desktop.Analyzers assembly.